test/2-dynLib-dep/dub.json

```diff
@@ -3,5 +3,7 @@
     "dependencies": {
         "dynlib-simple": { "path": "../1-dynLib-simple/" }
     },
+    "lflags-linux": ["-rpath", "$$ORIGIN"],
+    "lflags-darwin": ["-rpath", "@executable_path"],
```

Comment:
I'm not sure I understand where `$ORIGIN` comes from ?

Reply:
It's a special placeholder for the executable's directory, and is added as-is as `DT_RUNPATH` to the executable.